pkg/kubectl/cmd/config/set_cluster.go

```diff
@@ -113,7 +116,7 @@ func (o createClusterOptions) run() error {
 }
 
 // cluster builds a Cluster object from the options
-func (o *createClusterOptions) modifyCluster(existingCluster clientcmdapi.Cluster) clientcmdapi.Cluster {
+func (o *createClusterOptions) modifyCluster(existingCluster *clientcmdapi.Cluster) (cluster *clientcmdapi.Cluster, err error) {
```

Comment:
change the arg and return value back to non-pointers... the intent was to make this not mutate the param